bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerBaseHandler.java

```diff
@@ -69,5 +69,31 @@ public void updateChannels() {
         }
     }
 
+    @SuppressWarnings("null")
+    protected void handleNextDurationCommand(ChannelUID channelUID, Command command) {
+        if (!(command instanceof QuantityType<?>)) {
+            logger.info("Ignoring implausible non-QuantityType command for NEXT_DURATION");
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be warn.

Reply:
Done, changed to warn as suggested.

Reply:
FIXED: I had to roll back to fix some merge conflicts when rebasing and this must have been undone. Thanks.